// error line: 4

Main (->)
    def a <- [1, true]
end
